README.md

```diff
@@ -100,7 +103,8 @@ Which one of the following concepts does this best illustrate?
 What is the `ui-sref` directive, and how is it used?
 
 ```text
-Your answer goes here...
+"ui-sref" is link helper method in angular. It is used in an <a> tag and view page must be passed inside the method.
+
 ```
 
```

Comment:
You would use it instead of an `<a>` tag, and is for jumping to a different state (which wouldn't trigger any page refreshes or navigating elsewhere like a link would)